compiler/rustc_llvm/llvm-wrapper/RustWrapper.cpp

```diff
@@ -1528,6 +1528,21 @@ extern "C" LLVMValueRef LLVMRustGetInstrProfIncrementIntrinsic(LLVMModuleRef M)
               (llvm::Intrinsic::ID)llvm::Intrinsic::instrprof_increment));
 }
 
+extern "C" LLVMValueRef LLVMRustGetInstrProfMCDCParametersIntrinsic(LLVMModuleRef M) {
+  return wrap(llvm::Intrinsic::getDeclaration(unwrap(M),
+              (llvm::Intrinsic::ID)llvm::Intrinsic::instrprof_mcdc_parameters));
+}
+
+extern "C" LLVMValueRef LLVMRustGetInstrProfMCDCTVBitmapUpdateIntrinsic(LLVMModuleRef M) {
+  return wrap(llvm::Intrinsic::getDeclaration(unwrap(M),
+              (llvm::Intrinsic::ID)llvm::Intrinsic::instrprof_mcdc_tvbitmap_update));
+}
+
+extern "C" LLVMValueRef LLVMRustGetInstrProfMCDCCondBitmapIntrinsic(LLVMModuleRef M) {
+  return wrap(llvm::Intrinsic::getDeclaration(unwrap(M),
+              (llvm::Intrinsic::ID)llvm::Intrinsic::instrprof_mcdc_condbitmap_update));
+}
+
```

Comment:
I noticed that these casts to `(llvm::Intrinsic::ID)` should be unnecessary, so I filed #123591 to get rid of them from the existing `LLVMRustGetInstrProfIncrementIntrinsic` function.

Also, these functions will probably need some kind of `#if LLVM_VERSION_GE(18, 0)` so that the missing intrinsics don't break the build on LLVM 17.